Dockerfile

```diff
@@ -0,0 +1,54 @@
+
+FROM openjdk:8-jre-alpine
+
+# Expose ports
+EXPOSE 80
+
+# Add tags
+LABEL \
+  maintainer="The Petals team" \
+  contributors="Pierre Souquet (Linagora), Vincent Zurczak (Linagora)" \
+  github="https://github.com/petalslink"
+
+# Build arguments
+ARG CLI_VERSION="LATEST"
+ARG MAVEN_POLICY="releases"
+ARG BASE_URL="https://repository.ow2.org/nexus/service/local/artifact/maven/redirect"
+
+ARG CLI_HOST="localhost"
```

Comment:
This should not be a build argument.
It must be an environement variable, used only when we launch a container from the image. Not when we build the image.

Reply:
It is both, you can set it either at build or run, run with prevail.
That way we could have an easy to run, pre-set demo (with maven artifacts). I'll remove it.
But should I really keep `DEPLOY_FILE` and `DEPLOY_URLS` as argument then ? It does not make any sense to keep them now.

Reply:
We will build only one image per version.
`DEPLOY_FILE` and `DEPLOY_URLS` are not build arguments, but run arguments. When we instantiate the image. :ice_skate:

Reply:
It was done the same way as for CLI_HOST CLI_PORT etc ...
I'm removing the argument and leaving only the environment variable.